_layouts/home.html

```diff
@@ -3,24 +3,28 @@
 ---
 
 <div class="home">
+  {% if page.title %}
+    <h1 class="page-heading">{{ page.title }}</h1>
+  {% endif %}
 
   {{ content }}
 
-  <h1 class="page-heading">Posts</h1>
-  
-  <ul class="post-list">
-    {% for post in site.posts %}
-      <li>
-        {% assign date_format = site.minima.date_format | default: "%b %-d, %Y" %}
-        <span class="post-meta">{{ post.date | date: date_format }}</span>
+  {% if site.posts.size > 0 %}
+  <h2 class="post-list-heading">{{ page.list_title | default: "Posts" }}</h2>
```

Comment:
What if the user wanted the list heading to say `All posts` or Posts in another language? Wouldn't it be better entering a front matter key instead of maintaining an entire `home.html` at `source` just to edit this one line?